spec/statelint_spec.rb

```diff
@@ -402,15 +402,15 @@
     expect(problems.size).to eq(0)
   end
 
-  it 'should allow ItemProcessor in Map' do
```

Comment:
Should we not keep the test case with valid definition with only `ItemProcessor`? We should add these tests in addition to the existing one instead of replacing it.

Reply:
Good catch! That was deleted by mistake